src/keystone.nim

```diff
@@ -92,7 +92,7 @@ type
   ##  Resolver callback to provide value for a missing symbol in @symbol.
   ##  To handle a symbol, the resolver must put value of the symbol in @value,
   ##  then returns True.
-  SymbolResolver* = proc (symbol: cstring; value: ptr csize): bool {.cdecl.}
+  SymbolResolver* = proc (symbol: cstring; value: ptr csize_t): bool {.cdecl.}
```

Comment:
I wasn't aware of the difference between `csize` and `csize_t`. Thanks for the fix.

Reply:
its just changed recently, and `csize` is deprecated